packages/flutter_tools/lib/src/ios/device_ios.dart

```diff
@@ -235,7 +235,7 @@ class IOSDevice extends Device {
       return 2;
     }
     return await runCommandAndStreamOutput([loggerPath],
-        prefix: 'iOS dev: ', filter: new RegExp(r'.*SkyShell.*'));
+        prefix: 'iOS: ', filter: new RegExp(r'.*FlutterRunner.*'));
```

Comment:
The `filter` named argument expects a `RegExp`. So I couldn't specify a string. Deleted the `.*` though.